app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError(
+            "Invalid command format. "
+            'Expected format: "cp <source_file> <destination_file>"')
+    source_file = parts[1]
```

Comment:
Use unpacking to get ` source_file and destination_file` - it would be more pythonic approach

Reply:
Did you change this comment?